src/languages/en.js

```diff
@@ -808,12 +808,23 @@ export default {
         title: 'Request a call',
         subtitle: 'Have questions, or need help?',
         description: 'Our team of guides are on hand to help you each step of the way. Type in your name and phone number, and we’ll give you a call back.',
+        extension: 'Extension (Optional)',
         callMe: 'Call me',
         growlMessageOnSave: 'Call requested.',
         growlMessageEmptyName: 'Please provide both a first and last name so our guides know how to address you!',
         growlMessageNoPersonalPolicy: 'I wasn\'t able to find a personal policy to associate this Guides call with, please check your connection and try again.',
         callButton: 'Call',
         callButtonTooltip: 'Get live help from our team',
+        waitTime: {
+            calculating: 'Calculating wait time...',
+            fiveHoursPlus: 'The current wait time is longer than 5 hours.',
+            hoursAndMinutes: ({minutes}) => `The current wait time is ${Math.floor(minutes / 60)} hours and ${minutes % 60} minutes. `,
+            minutes: ({minutes}) => `The current wait time is ${minutes} minutes. `,
+            guides: 'Please note that our Guides are typically available from Sunday at 5pm CT to Friday at 5pm CT.',
+        },
+        error: {
+            phoneExtension: 'Please enter a valid phone extension number',
+        },
```

Comment:
It's interesting that for the other fields we do a growl if they are empty instead of a form error. It could be good as a follow up to make that consistent.

Reply:
If you just post in #expensify-open-source a contributor can do it. Or you could do it yourself if you wanted.

Reply:
Should we use a form error for the first and last name fields? The growl error goes away very quickly. I would be happy to fix that in another issue.

Reply:
Yeah I think we should. Just like we do for the other two fields.